RemoteSignTool/RemoteSignTool.Server/Services/SignToolService.cs

```diff
@@ -1,11 +1,19 @@
-﻿using System.IO;
+﻿using NLog;
```

Comment:
I'm sorting usings by System ones first (can be configured in Tools -> Options -> Text Editor -> C# -> Advanced -> Place 'System' usings first when sorting usings.